dbgpt/model/adapter/embeddings_loader.py

```diff
@@ -50,6 +50,14 @@ def load(self, model_name: str, param: BaseEmbeddingModelParameters) -> Embeddin
                 if proxy_param.proxy_backend:
                     openapi_param["model_name"] = proxy_param.proxy_backend
                 return OpenAPIEmbeddings(**openapi_param)
+            elif model_name in ["proxy_tongyi"]:
+                from dbgpt.rag.embedding import TongYiEmbeddings
+
+                tongyi_param = {}
```

Comment:
The `API_KEY` should be `TONGYI_PROXY_API_KEY`

Reply:
I have change the code. use CFG.tongyi_proxy_api_key to be the api_key